src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -311,23 +310,27 @@ class ImageBaseViewer extends BaseViewer {
     }
 
     /**
-     * Handles image element loading errors.
+     * Handles a content download error
      *
-     * @private
-     * @param {Error} err - Error to handle
+     * @param {Error} err - Load error
+     * @param {string} imgUrl - URL we are using as the image src
      * @return {void}
      */
-    errorHandler(err) {
-        /* eslint-disable no-console */
-        console.error(err);
-        /* eslint-enable no-console */
-
-        // Display a generic error message but log the real one
-        const error = err;
-        if (err instanceof Error) {
-            error.displayMessage = __('error_refresh');
+    handleDownloadError(err, imgUrl) {
```

Comment:
This is the same as `ImageBaseViewer.handleDownloadError()`. Can you make this a function in the parent class?

Reply:
Do you mean in docBase vs. imageBase? The doc version of this function throws a different error (error_document vs. error_refresh, I need to fix that), and other viewers will want custom behavior here as well. See how the dash-viewer handles download errors currently.

Reply:
If the only difference between this and docBase is the error message, wouldn't it make sense to use this as the default and maybe pass in an error message in each viewer? The rest of the code seems to stay the same

Reply:
Looking at it again you're right, the only significant difference are the messages, so I'll pass those in.